docs/features/multi-env.mdx

```diff
@@ -5,4 +5,14 @@ icon: "rectangle-history-circle-plus"
 
 <Snippet file="cloud/cloud-feature-tag.mdx" />
 
-Under construction 🚧
+An environment is an Elementary schema in a data warehouse created by the **[Elementary dbt package](https://docs.elementary-data.com/cloud/onboarding/quickstart-dbt-package).**
+
+A company that has only one dbt project likely has two environments - one for development and the other for production. However, larger organizations may have several dbt projects in different data warehouses, which means they have multiple environments. With Elementary Cloud, monitoring the data quality of all these environments from a single place becomes easy.
+
+<Frame>
+  <img
+    src="https://res.cloudinary.com/diuctyblm/image/upload/v1701767145/elementary_multiple_environments_szdsfj.gif"
```

Comment:
I think what I'd show in the GIF here is the results across environments and how you can see / configure all of them, not how to connect a new environment. The purpose being to show the value of having multiple envs configured in one place. WDYT?